lib/rspec/matchers/english_phrases.rb

```diff
@@ -0,0 +1,58 @@
+module RSpec
+  module Matchers
+    # Facilitates converting ruby objects to English phrases.
+    module EnglishPhrases
```

Comment:
I think I slightly prefer `EnlighsPhrasing` over `EnglishPhrases` (hard to explain why, though, and it's a very lightly helped opinion).